src/vs/editor/contrib/dropOrPasteInto/browser/copyPasteController.ts

```diff
@@ -282,6 +282,12 @@ export class CopyPasteController extends Disposable implements IEditorContributi
 					return;
 				}
 
+				// If the PasteLinkEditProvider does not return anything, use the default paste handler
+				if (supportedProviders[0].pasteMimeTypes?.length === 1 && supportedProviders[0].pasteMimeTypes[0] === 'text/plain' && providerEdits.length === 1 && providerEdits[0].id === 'text') {
+					await this.applyDefaultPasteHandler(dataTransfer, metadata, tokenSource.token);
+					return;
+				}
+
```

Comment:
Wondering if we could get away with simplifying this condition to match the one on line 274 above? I did a little testing and this seems to also address https://github.com/microsoft/vscode/issues/186343, but please do some testing before accepting this suggestion :) 

```suggestion
				if (providerEdits.length === 1 && providerEdits[0].id === 'text') {
					// Only our default text provider returned edits
					await this.applyDefaultPasteHandler(dataTransfer, metadata, tokenSource.token);
					return;
				}
```